### DISCUSSION TITLE
Tiering in one volume server.

### DISCUSSION THREAD

**Discussion by ekozlov-search** [seaweedfs/seaweedfs#2407] [Q&A]
I have one volume server with two disk: hdd and ssd.
When I try to move collection from one dist to another using volume.tier.move - I have error. Am I doing something wrong?

> volume.tier.move -fromDiskType=fast -toDiskType=slow -collectionPattern=test1 -fullPercent=0.001 -quietFor=1s -force
collect fast volumes quiet for: 1 seconds
tier move volumes: [5 6 7 1 2 3 4]
can not find disk type slow for volume 5
can not find disk type slow for volume 6
can not find disk type slow for volume 7
can not find disk type slow for volume 1
can not find disk type slow for volume 2
can not find disk type slow for volume 3
can not find disk type slow for volume 4

![image](https://user-images.githubusercontent.com/61773005/139661514-32f72be2-ee27-4115-8eb1-d580884445d1.png)

**Comment by chrislusf** [comment]
Current code assumes the destination is a different volume server.

**Comment by ekozlov-search** [comment]
Is this functionality possible?

**Reply by chrislusf** [reply]
Going through network to copy to itself seems unnecessarily costly. But adding a shortcut to optimize for this adds some complexity but not really very useful if there is only one volume server.
